microblog/src/lib/server/auth.ts

```diff
@@ -0,0 +1,21 @@
+import { PrismaAdapter } from "@lucia-auth/adapter-prisma";
+import { Lucia } from 'lucia';
+import db from "./db";
+import { dev } from "$app/environment";
+import { decl } from "postcss";
+
+const adapter = new PrismaAdapter(db.session, db.user);
```

Comment:
`PrismaClient.session` doesn't exist anymore? so this just errors out. not fun, not documented

Reply:
lmao you forgot to add the session model dumbass